wiki/Reporting_bad_behaviour/de.md

```diff
@@ -14,7 +9,7 @@ Versehentliche Meldungen werden ignoriert. Benutzer, die absichtlich invalide Me
 | Art des Verstoßes | Kontaktperson |
 | :-- | :-- |
 | Offensive private Nachrichten | **Keine**: [ignoriere den Benutzer](/wiki/Chat_Console#commands-list) stattdessen. Für einen dauerhaften Effekt muss die Ignorieren-Liste in den [Optionen](/wiki/Options) des Spiel-Clients genutzt werden. |
-| Verstöße bezogen auf Modding und/oder Mapping | NAT, via der [NAT-/BN-Management-](https://bn.mappersguild.com/reports)-Webseite |
+| Verstöße bezogen auf Modding und/oder Mapping | [NAT](/wiki/People/The_Team/Nomination_Assessment_Team) oder Mitglieder des [GMT](/wiki/People/The_Team/Global_Moderation_Team) mit Schwerpunkt auf Mapping/Modding-Community-Moderation, via der [NAT-/BN-Management-](https://bn.mappersguild.com/reports)-Webseite |
```

Comment:
```suggestion
| Verstöße bezogen auf Modding und/oder Mapping | [NAT](/wiki/People/The_Team/Nomination_Assessment_Team) oder Mitglieder des [GMTs](/wiki/People/The_Team/Global_Moderation_Team) mit Schwerpunkt auf Mapping/Modding-Community-Moderation, via der [NAT-/BN-Management](https://bn.mappersguild.com/reports)-Webseite |
```
First one: It is `Mitglieder des Teams` if you simplify it. So, there should be a `s` at the end
Second one: Just noticed that there's one line too much (`--`)